pkg/model/bootstrapscript.go

```diff
@@ -282,47 +282,34 @@ func (b *BootstrapScript) createProxyEnv(ps *kops.EgressProxySpec) string {
 			httpProxyURL += ps.HTTPProxy.Host
 		}
 
-		// Set base env variables
-		buffer.WriteString("export http_proxy=" + httpProxyURL + "\n")
-		buffer.WriteString("export https_proxy=${http_proxy}\n")
-		buffer.WriteString("export no_proxy=" + ps.ProxyExcludes + "\n")
-		buffer.WriteString("export NO_PROXY=${no_proxy}\n")
-
-		// TODO move the rest of this configuration work to nodeup
-
-		// Set env variables for docker
-		buffer.WriteString("echo \"export http_proxy=${http_proxy}\" >> /etc/default/docker\n")
-		buffer.WriteString("echo \"export https_proxy=${http_proxy}\" >> /etc/default/docker\n")
-		buffer.WriteString("echo \"export no_proxy=${no_proxy}\" >> /etc/default/docker\n")
-		buffer.WriteString("echo \"export NO_PROXY=${no_proxy}\" >> /etc/default/docker\n")
-
 		// Set env variables for base environment
-		buffer.WriteString("echo \"export http_proxy=${http_proxy}\" >> /etc/environment\n")
-		buffer.WriteString("echo \"export https_proxy=${http_proxy}\" >> /etc/environment\n")
-		buffer.WriteString("echo \"export no_proxy=${no_proxy}\" >> /etc/environment\n")
-		buffer.WriteString("echo \"export NO_PROXY=${no_proxy}\" >> /etc/environment\n")
-
-		// Set env variables to systemd
-		buffer.WriteString("echo DefaultEnvironment=\\\"http_proxy=${http_proxy}\\\" \\\"https_proxy=${http_proxy}\\\"")
-		buffer.WriteString("echo DefaultEnvironment=\\\"http_proxy=${http_proxy}\\\" \\\"https_proxy=${http_proxy}\\\"")
-		buffer.WriteString(" \\\"NO_PROXY=${no_proxy}\\\" \\\"no_proxy=${no_proxy}\\\"")
-		buffer.WriteString(" >> /etc/systemd/system.conf\n")
+		buffer.WriteString("echo \"export http_proxy=" + httpProxyURL + "\" >> /etc/environment\n")
```

Comment:
perhaps using `` so you don't need to blackslash

Reply:
Yeah I did think initially about doing this, all the backslashes make it difficult to read. I'll fix up and test..